pkg/cluster/sync.go

```diff
@@ -695,13 +695,11 @@ func (c *Cluster) syncPreparedDatabases() error {
 	for preparedDbName, preparedDB := range c.Spec.PreparedDatabases {
 		if err := c.initDbConnWithName(preparedDbName); err != nil {
 			return fmt.Errorf("could not init connection to database %s: %v", preparedDbName, err)
+		} else {
+			c.logger.Debugf("opened preparedDbName: %s", preparedDbName)
```

Comment:
same as for closing: I think this log line is not needed

Reply:
We can remove, but this info was very valuable in figuring out what actions does OPR do automatically when handling preparedDbs.

Reply:
I can image that it helped solving the issue, but the operator opens and closes connections several times. We shouldn't log that. It should be enough to have this extra debug line `syncing prepared database ...`.